library/core/src/ptr/const_ptr.rs

```diff
@@ -1285,7 +1288,7 @@ impl<T: ?Sized> *const T {
     /// [`ptr::copy_nonoverlapping`]: crate::ptr::copy_nonoverlapping()
     #[rustc_const_stable(feature = "const_intrinsic_copy", since = "1.63.0")]
     #[stable(feature = "pointer_methods", since = "1.26.0")]
-    #[inline]
+    #[inline(always)]
```

Comment:
Same deal. Our inline cost model doesn't make trivial "this just calls another function" wrappers good candidates for inline(always), if that function may be big.

And `copy_nonoverlapping` definitely can be pretty big if `T` or `count` are big.

So this shouldn't be inline(always).